native-sdk/src/modules/royalty/royalty.rs

```diff
@@ -0,0 +1,31 @@
+use radix_engine_interface::api::node_modules::royalty::{
+    ComponentRoyaltyCreateInput, COMPONENT_ROYALTY_BLUEPRINT, COMPONENT_ROYALTY_CREATE_IDENT,
+};
+use radix_engine_interface::api::types::RoyaltyConfig;
+use radix_engine_interface::api::ClientApi;
+use radix_engine_interface::constants::ROYALTY_PACKAGE;
+use radix_engine_interface::data::scrypto::model::Own;
+use radix_engine_interface::data::scrypto::*;
+use sbor::rust::prelude::*;
+
+pub struct ComponentRoyalty;
+
+impl ComponentRoyalty {
+    pub fn sys_create<Y, E: Debug + ScryptoDecode>(
+        api: &mut Y,
```

Comment:
Can we have the api be the last parameter for consistency sake?

Reply:
Changed this in the Package Object clean-up PR. :)